product_variant_sale_price/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': 'Product Variant Sale Price',
+    'summary': 'Allows to write fixed prices in product variants',
+    'version': '11.0.1.0.0',
+    'category': 'Product Management',
+    'website': 'https://odoo-community.org/',
```

Comment:
This should be https://github.com/OCA/product-variant
Take a look in: https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10